helm/templates/bootstrap-sentinel.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
```

Comment:
This should be a Job following last week's discussion, right?

Reply:
If we change it to a Job, then the job won't disappear and we have the same problem, just with another resource. Additionally, the pod now gets deleted anyways by the webhook-creation-job, so no need to change it anymore.